src/domain/event/hooks/useRelatedEvents.ts

```diff
@@ -0,0 +1,37 @@
+import { useApolloClient } from '@apollo/client';
+import React from 'react';
+
+import { EventFieldsFragment } from '../../../generated/graphql';
+import { getRelatedEvents } from '../utils';
+
+const useRelatedEvents = (event: EventFieldsFragment) => {
```

Comment:
Small explanation what is considered relatedEvent might help other developers in the future

Reply:
Added comment for the hook. I hope it's clearer now